lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -285,6 +285,40 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc """
+  Same as `start_supervised/2` but returns the PID on success and raises if
+  not started porperly.
+  """
+  @spec start_supervised!(Supervisor.child_spec | module | {module, term}, keyword) :: pid
+  def start_supervised!(child_spec_or_module, opts \\ []) do
+    case start_supervised(child_spec_or_module, opts) do
+      {:ok, pid} ->
+        pid
+      {:ok, pid, _info} ->
```

Comment:
According to the [spec](https://hexdocs.pm/elixir/Supervisor.html#t:on_start_child/0), the [`Supervisor.start_child/2`](https://hexdocs.pm/elixir/Supervisor.html#start_child/2) return tuple can be like this also.

Reply:
Yes, beautiful.